algorithms/algebraic/discrete_log/discrete_log.ipynb

```diff
@@ -85,20 +85,20 @@
     "\n",
     "@qfunc\n",
     "def discrete_log_oracle(\n",
-    "    g: CInt,\n",
-    "    x: CInt,\n",
-    "    N: CInt,\n",
+    "    g_generator: CInt,\n",
+    "    x_logarithm: CInt,\n",
```

Comment:
Notice that x is not the logarithm, but a group element.
I suggest `x_element` instead

Reply:
Thank You for your suggestion. I will update it soon